src/main/java/core/basesyntax/Employee.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Employee implements SalaryCalculator {
+    public static final int STARTING_SALARY = 0;
+    private final String name;
+    private int currentSalary;
+
+    public Employee(String name) {
+        this.name = name;
+        this.currentSalary = STARTING_SALARY;
```

Comment:
`int` variables equals `0` by default, remove this constant and `currentSalary` initialization in constructor.